packages/react-web-cli/src/AblyCliTerminal.tsx

```
@@ -888,14 +1323,655 @@
+          debugLog(`[AblyCLITerminal] [Secondary] Received PTY data: "${sanitizedData}"`);
+          
+          if (secondaryPtyBuffer.current.length > MAX_PTY_BUFFER_LENGTH) {
+            secondaryPtyBuffer.current = secondaryPtyBuffer.current.slice(secondaryPtyBuffer.current.length - MAX_PTY_BUFFER_LENGTH);
+          }
+          
+          // Strip ANSI codes
+          const cleanBuf = secondaryPtyBuffer.current.replace(/\u001B\[[0-9;]*[mGKHF]/g, '');
+          
+          // Only detect the prompt if it appears at the end of the buffer
+          if (TERMINAL_PROMPT_PATTERN.test(cleanBuf)) {
+            debugLog('[AblyCLITerminal] [Secondary] Shell prompt detected at end of buffer');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)